tests/compute_transaction_test.py

```diff
@@ -0,0 +1,127 @@
+
+killWallet=not dontKill
+killEosInstances=not dontKill
+if nodesFile is not None:
+    killEosInstances=False
+
+Utils.Debug=debug
+testSuccessful=False
+
+random.seed(seed) # Use a fixed seed for repeatability.
+cluster=Cluster(walletd=True)
+
+walletMgr=WalletMgr(True)
+EOSIO_ACCT_PRIVATE_DEFAULT_KEY = "5KQwrPbwdL6PhXujxW37FSSQZ1JiwsST4cqQzDeyXtP79zkvFD3"
+EOSIO_ACCT_PUBLIC_DEFAULT_KEY = "EOS6MRyAjQq8ud7hVNYcfnVPJqcVpscN5So8BhtHuGYqET5GDW5CV"
+
+try:
+    if dontLaunch: # run test against remote cluster
+        jsonStr=None
+        with open(nodesFile, "r") as f:
+            jsonStr=f.read()
+        if not cluster.initializeNodesFromJson(jsonStr):
+            errorExit("Failed to initilize nodes from Json string.")
+        total_nodes=len(cluster.getNodes())
+
+        walletMgr.killall(allInstances=killAll)
+        walletMgr.cleanup()
+        print("Stand up walletd")
+        if walletMgr.launch() is False:
+            errorExit("Failed to stand up keosd.")
+        else:
+            cluster.killall(allInstances=killAll)
+            cluster.cleanup()
+
+    Print ("producing nodes: %s, non-producing nodes: %d, topology: %s, delay between nodes launch(seconds): %d" % (pnodes, total_nodes-pnodes, topo, delay))
+
+    Print("Stand up cluster")
+    if cluster.launch(pnodes=pnodes, totalNodes=total_nodes, topo=topo, delay=delay) is False:
+       errorExit("Failed to stand up eos cluster.")
+
+    Print ("Wait for Cluster stabilization")
+    # wait for cluster to start producing blocks
+    if not cluster.waitOnClusterBlockNumSync(3):
+     errorExit("Cluster never stabilized")
+
+    Print("Creating account1")
+    account1 = Account('account1')
+    account1.ownerPublicKey = EOSIO_ACCT_PUBLIC_DEFAULT_KEY
+    account1.activePublicKey = EOSIO_ACCT_PUBLIC_DEFAULT_KEY
+    cluster.createAccountAndVerify(account1, cluster.eosioAccount, stakedDeposit=1000)
+
+    Print("Creating account2")
+    account2 = Account('account2')
+    account2.ownerPublicKey = EOSIO_ACCT_PUBLIC_DEFAULT_KEY
+    account2.activePublicKey = EOSIO_ACCT_PUBLIC_DEFAULT_KEY
+    cluster.createAccountAndVerify(account2, cluster.eosioAccount, stakedDeposit=1000)
+
+    Print("Validating accounts after bootstrap")
+    cluster.validateAccounts([account1, account2])
+
+    node = cluster.getNode()
+    transferAmount="1000.0000 {0}".format(CORE_SYMBOL)
+
+    node.transferFunds(cluster.eosioAccount, account1, transferAmount, "fund account")
+    preBalances = node.getEosBalances([account1, account2])
+    Print("Starting balances:")
+    Print(preBalances)
+
+    txnData ='{"from":"account1","to":"account2","quantity":"5.0000 EOS","memo":"tx1"}'
+
+    Print("Sending read-only transfer")
+    trx = {
+        "actions": [{"account": "eosio.token", "name": "transfer",
+        "authorization": [{"actor": account2.name, "permission": "active"}],
+        "data": txnData}]
+    }
+
+    node.pushTransaction(trx, opts='--read-only', permissions=account2.name)
```

Comment:
Should verify this returns True, otherwise it might have failed and your test would still pass.